src/components/Image/Image.tsx

```diff
@@ -177,6 +179,7 @@ export const Image = chakra(
         loading={isLazy ? "lazy" : undefined}
         {...srcProp}
         __css={{ ...styles.img, ...additionalImageStyles }}
+        {...rest}
```

Comment:
This is a very logical chage.  As a dev, I would assume any props that I was passing in would end up on the `img` itself.  One step at a time, right?